host-bin/startkodi

```diff
@@ -14,5 +14,10 @@ By default, it will log into the primary user on the first chroot found.
 
 Options are directly passed to enter-chroot; run enter-chroot to list them."
 
+# Forward ports needed to browse smb shares
+if test -n "$(ip -4 -o addr show dev wlan0| awk '{split($4,a,"/");print a[1]}')"; then
```

Comment:
This would assume you are using wlan0, you also could be using a usb ethernet adapter or another wlan device.

Reply:
running that command returns a "1"

i agree that we should take into consideration other devices, i just don't know how...

Reply:
@zguithues hmm strange I just tried it on some different linux devices and my chromebook and that seems to work. What do you get when you run `ip route get 1` for output. Maybe the awk part isn't full proof. If not, there sure will be other ways to get the used ip-address, without looking at the device.